internal/cmd/config.go

```diff
@@ -432,9 +432,42 @@ func Configure(v *viper.Viper, _ *pflag.FlagSet) {
 	v.SetDefault("cluster::autoscale::hpa::prometheus::serviceName", "monitor-prometheus-server")
 	v.SetDefault("cluster::autoscale::hpa::prometheus::serviceContext", "prometheus")
 	v.SetDefault("cluster::autoscale::hpa::prometheus::localPort", 9090)
-	v.SetDefault("cluster::autoscale::charts::clusterAutoscaler::chart", "banzaicloud-stable/cluster-autoscaler")
-	v.SetDefault("cluster::autoscale::charts::clusterAutoscaler::version", "0.12.3")
+	v.SetDefault("cluster::autoscale::charts::clusterAutoscaler::chart", "stable/cluster-autoscaler")
+	v.SetDefault("cluster::autoscale::charts::clusterAutoscaler::version", "6.2.0")
 	v.SetDefault("cluster::autoscale::charts::clusterAutoscaler::values", map[string]interface{}{})
+	v.SetDefault("cluster::autoscale::charts::clusterAutoscaler::imageVersions", []interface{}{
+		map[string]interface{}{
+			"k8sVersion": "<=1.12.x",
```

Comment:
These are called (version)constraints, not versions.

Reply:
could be imageVersionConstraints but seems a bit too long to me, or could be just constraints?

Reply:
renamed to imageVersionConstraints